README.md

```diff
@@ -26,15 +29,6 @@ Each cloud-enabled hub requires the following AWS resources:
     - A _trust policy_ that stipulates the role can only be used by GitHub Actions that originate from the main branch of the hub's repository.
     - A _permission policy_ that grants write access to the hub's S3 bucket.
 
-This repository contains two Pulumi-related GitHub workflows create and destroy the above AWS resources as needed:
```

Comment:
Moved this info down to a dedicated section for "permissions" and removed the OIDC image, which was muddying the waters, imo (we're using OIDC for AWS auth, but it's not the main focus of what the codebase is doing)